packages/flutter_test/lib/src/controller.dart

```diff
@@ -679,4 +692,11 @@ class LiveWidgetController extends WidgetController {
     binding.scheduleFrame();
     await binding.endOfFrame;
   }
+
+  @override
+  Future<List<Duration>> handlePointerEventRecord(List<PointerEventRecord> records) {
+    // TODO(CareF): This will be implemented after we decide what should be the
+    // correct pumping strategy.
```

Comment:
please file a bug and link it here

Reply:
I'm thinking implementing it should be mostly the same as the implementation in `WidgetTester`, but when I had a look at how we are using this `LiveWidgetController` in the repo I found very limited to a few test cases in `dev/`, where
1. `dev/benchmarks/complex_layout/test_memory/scroll_perf.dart` is using `fling` which already is pumping regardless. 
2. `dev/integration_tests/flutter_gallery/test/live_smoketest.dart` and `dev/integration_tests/flutter_gallery/test_memory/image_cache_memory.dart` don't pump at all
3. `packages/flutter_driver/lib/src/extension/extension.dart` is used in `flutter_driver` and don't have pump. 

From these use cases I believe 1 is just not taking much consideration about pumping while 2 and 3 already suggest not to pump.

Reply:
And `LiveWidgetController` is not unit tested.